README.md

```diff
@@ -1,23 +1,27 @@
 # Calico Networking for rkt
 
 ## Building the plugin
-To build the calico-kubernetes plugin, clone this repository and run `make`.  This will build the binary, as well as run the unit tests.  To just build the binary, with no tests, run `make binary`.  To only run the unit tests, simply run `make ut`.
+
+To build the Calico Networking Plugin for Rkt, clone this repository and run `make`.  This will build the binary, as well as run the unit tests.  To just build the binary, with no tests, run `make binary`.  To only run the unit tests, simply run `make ut`.
 
 ## Using rkt Plugins
+
 The [documentation](https://github.com/coreos/rkt/blob/master/Documentation/networking.md) for Network Plugins in rkt will walk you through the basics of setting up networking in rkt.
 
 ## Requirements
+
 * A working [etcd](https://github.com/coreos/etcd) service
-* Though Calico is capable of networking rkt containers, our core software is distributed and deployed in a [docker container](https://github.com/projectcalico/calico-docker/blob/master/docs/getting-started/default-networking/Demonstration.md). For now, you will need to run calico in docker before starting your rkt contianers. This can be easily done by downloading the [calicoctl binary](https://github.com/projectcalico/calico-docker/releases) and running the following command: `calicoctl node --ip=$IP`
+* Though Calico is capable of networking rkt containers, our core software is distributed and deployed in a [docker container](https://github.com/projectcalico/calico-docker/blob/master/docs/getting-started/default-networking/Demonstration.md). For now, you will need to run calico in docker before starting your rkt contianers. This can be easily done by downloading the [calicoctl binary](https://github.com/projectcalico/calico-docker/releases) and running the following command: `sudo calicoctl node --ip=<IP> --rkt`
 
 ## Installing
-* Move the binary to the rkt plugin directory.
+
+* The `--rkt` flag of the `calicoctl node` command will install the plugin for you. Alternatively you can install the [binary](https://github.com/projectcalico/calico-rkt/releases/) yourself and move it to the rkt plugin directory.
```

Comment:
Seems a little backwards to have the flag listed before the command. I'd personally say to run the `calicoctl node` command with the `--rkt` flag to install the plugin.